src/Configuration/src/SpringBootEnvBase/SpringBootEnvExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using System;
+
+namespace Steeltoe.Extensions.Configuration.SpringBootEnv
+{
+    public static class SpringBootEnvExtensions
+    {
+        /// <summary>
+        /// Add a SpringBootEnv configuration source to the <see cref="ConfigurationBuilder"/>.
+        /// This Source translates configuration keys in '.' delimited style to a format understood by .NET
+        /// </summary>
+        /// <param name="builder">the configuration builder</param>
+        /// <param name="loggerFactory">the logger factory to use</param>
```

Comment:
I don't think we need to log anything in this provider, it could cause confusion if we allow passing in a loggerfactory that never does anything